extensions/amp-story/1.0/amp-story-quiz.js

```diff
@@ -102,6 +132,29 @@ export class AmpStoryQuiz extends AMP.BaseElement {
     createShadowRootWithStyle(this.element, this.quizEl_, CSS);
   }
 
+  /** @override */
+  layoutCallback() {
+    this.responseDataPromise_ = this.retrieveReactionData_();
+  }
+
+  /**
+   * Gets a Promise to return the unique AMP clientId
+   *
+   * @private
+   * @return {Promise<string>}
+   */
+  getClientId_() {
+    if (!this.clientIdPromise_) {
+      this.clientIdPromise_ = this.clientIdService_.then(data => {
+        return data.get(
+          {scope: 'amp-story', createCookieIfNotPresent: true},
+          Promise.resolve()
```

Comment:
Please add comment with the parameter so we know what it is and why we don't need it:
`data.get({...},  /* consent */ Promise.resolve())`